### PR TITLE
#303 better error message if repo does not have any commits

### LINKED ISSUE #303: mob start does not work on git repository without commits
When you clone an empty repository from git and want to start working on it the mob tool throws an error:

image

Currently I do not have a clue what a proper fix for that would be, since the mob tool would need to create a commit which is not developer made to create the corresponding branches.

Maybe it is not worth the effort investigating that.

### PR DESCRIPTION
**Pull request by hollesse**
closes #303

### CODE REVIEW

**Review by gregorriegler — COMMENTED** [review]
Is there a reason you chose a command that would yield an error, over a command that would give a successful result?
E.g. `git rev-list --all --count` would count all commits on all branches and yield 0 (no error)

**Comment by hollesse** [comment]
No that was just my knowledge. That's a good idea. Like it :)

**Comment by gregorriegler** [comment]
I made the change @hollesse, please review

File: mob.go [hunk 1]
```diff
@@ -276,10 +278,10 @@ func main() {
 	execute(command, parameters, configuration)
 }
 
-func hasNoCommits() bool {
-	_, _, err := runCommandSilent("git", "rev-list", "-n", "1", "HEAD")
-	return err != nil
-
+func hasCommits() bool {
+	commitCount := silentgit("rev-list", "--all", "--count")
+	println(commitCount)
```
Comment (hollesse) [inline mob.go:283]: Do we really like to print that? I don‘t think so. And if we would like to print it we should use the say function.

**Review by hollesse — COMMENTED** [review]
Except for the PrintLn it looks fine to me :)

File: mob.go (hunk 1 above)
Comment (gregorriegler) [reply mob.go:283]: its a mistake, thanks

Comment (gregorriegler) [reply mob.go]: fixed

**Pull request merged**